spec/requests/container_images_spec.rb

```diff
@@ -45,4 +45,55 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  context 'POST /api/container_images/scan' do
+    let(:provider) { FactoryGirl.create(:ems_kubernetes) }
+    let(:container_image) { FactoryGirl.create(:container_image, :ext_management_system => provider) }
+    let(:conatiner_id) { container_image.id }
+    let(:invalid_container_image) { container_image.id + 100 }
+    let(:invalid_image_url) { api_provider_container_image_url(nil, provider, invalid_container_image) }
```

Comment:
since this is only used here, maybe just use a bogus number instead, like 999,999